src/kernel/src/syscall/mod.rs

```diff
@@ -89,14 +89,15 @@ fn type_sys_kaction(
     lo: u64,
     arg: u64,
     _flags: u64,
+    arg2: u64,
 ) -> Result<KactionValue, KactionError> {
     let cmd = KactionCmd::try_from(cmd)?;
     let objid = if hi == 0 {
         None
     } else {
         Some(ObjID::new_from_parts(hi, lo))
     };
-    crate::device::kaction(cmd, objid, arg)
+    crate::device::kaction(cmd, objid, arg, arg2)
```

Comment:
Why are they arg, arg2 instead of the names below?

Reply:
As opposed to the arg5 names? kaction has 2 generic args vs. syscall's 6 generic args.

Reply:
Ah, OK, makes sense.  Every time I see something "generic" like that I get an itch at the back of my head :-)